.nucleus.yaml

```diff
@@ -26,13 +26,10 @@ branches:
     pull-request:
       <<: *branch-definition
       merge-method: force-push # release branch should always be in sync with master branch (linear history)
-  spring23:
-    pull-request:
-      <<: *branch-definition
-  summer23:
```

Comment:
I don't see a need to keep `summer23` around as all instances have already upgraded to `winter24`.